_posts/2021-12-14-announce-github-naming-change.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: "Announcing Github renaming change"
```

Comment:
Should we specify this is the branch being renamed? Just to any confusion for someone glancing at the title but not reading the full blog? Not that I've ever done something like that 🙂

Reply:
Changed title: "Announcing Github branch renaming change"